rero_mef/version.py

```diff
@@ -30,4 +30,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = '0.2.0'
+__version__ = '0.6.0'
```

Comment:
🤟🤟🤟 😂😂😂

Reply:
0.6.1 (or 0.7.0 soon), if the version must follow rero-ils release. In other case, I suggest 0.3.0

Reply:
Ok: 0.3.0